pysmt/test/test_solving.py

```diff
@@ -173,6 +173,9 @@ def test_examples_cvc4(self):
         for (f, validity, satisfiability, logic) in get_example_formulae():
             try:
                 if not logic.quantifier_free: continue
+                if logic.theory.arrays:
```

Comment:
This could be handled as a NoSolverAvailableError exception as CVC4 does not support (for now) arrays